content/en/docs/collector/trace-receiver.md

```diff
@@ -1918,6 +1918,11 @@ func NewSpanID() pcommon.SpanID {
 }
 ```
 
+> #### Check your work
+>
+> - Imported the `crypto/rand` package as `crand` to avoid name conflict with
+>   math/rand.
+
```

Comment:
It is my understanding that "check your work" sections highlight additions or changes that were mentioned previously.

In this case, there is no sample code that illustrates the import.

I'd suggest that we instead add the required import to the code block that ends at line 1919, and then drop this "Check your work" call out.

Reply:
yes, let's have that partial import

Reply:
@chalin please take another look